src/classes/AFFL_AccRecordType_TDTM.cls

```diff
@@ -146,9 +146,10 @@ public class AFFL_AccRecordType_TDTM extends TDTM_Runnable {
                     if (!containsNew(accsTypeChanged, accNewOldType.oldType)) {
                         contact.put(accNewOldType.oldType, null);
                     }
-                    contact.put(accNewOldType.newType, accNewOldType.accId);
-                    dmlWrapper.objectsToUpdate.add(contact);
-
+                    if (accNewOldType.newType!=NULL) {
+                        contact.put(accNewOldType.newType, accNewOldType.accId);
+                        dmlWrapper.objectsToUpdate.add(contact);
```

Comment:
@skosgi1 I just did a double-check on this and need to assign it back to you. Because the contact could be updated on line 147, we need to add the contact record to the dmlWrapper outside of this null check as was being done previously.

(This might affect W-033954?)

Reply:
Thanks for catching that @robertbwatson. You're right.